content/en/v3/admin/platforms/k3s/_index.md

```diff
@@ -38,6 +40,13 @@ export KUBECONFIG=~/.kube/config:~/.kube/k3s-config
 
 ```
 
+**For WSL2 users:**
+- Download k3s binary from https://github.com/rancher/k3s/releases/latest
+
+- `chmod +x k3s`
+
+- Run k3s `sudo ./k3s server`
```

Comment:
Make it clear that it needs to run in one terminal (In the past ppl get confused). Basically they should not kill the k3s server.

Reply:
![image](https://user-images.githubusercontent.com/40542456/164307661-96e9cade-1709-45cc-86cf-f8da84e4f780.png)
Do this look good?